shared/android/keybaselib/build-c-headers.sh

```diff
@@ -0,0 +1,22 @@
+# TODO make CLANG_HOME work on other platforms
+build() {
+    GOPATH=`pwd`/../go
+    CLANG_HOME=$ANDROID_HOME/ndk-bundle/toolchains/llvm/prebuilt/darwin-x86_64/bin
+    ARCH=$1
+
+    env \
+    GOOS=$2 \
+    GOARCH=$3 \
+    CGO_ENABLED=1 \
+    CC=$CLANG_HOME/$ARCH-clang \
+    CXX=$CLANG_HOME/$ARCH-clang++ \
+    go tool cgo -exportheader "c-headers/keybaselib-$3.h" ../../../go/bind/keybase.go
```

Comment:
This builds the c-headers from our go library. The linking is done in the cmakelist and aar exploding